R/agglomerate.R

```diff
@@ -45,26 +45,26 @@
 #'   to \code{getPrevalence} and \code{getPrevalentTaxa} and used in
 #'   \code{agglomeratebyPrevalence}
 #'   \itemize{
-#'        \item \code{remove_empty_ranks}: A single boolean value for selecting
+#'        \item \code{empty.ranks.rm}: A single boolean value for selecting
```

Comment:
I would use instead `rm.empty.ranks` (in line with the common parameter `rm.na`)

Reply:
By the way, could we just use `rm.na` here? Is there a real need for another argument name for removing NAs here?

Reply:
DO you mean `na.rm`?

na.rm is already in use in that function. Also empty.ranks.rm just tidies up the output rowData which makes it less important parameter

Reply:
Ah right, na.rm - I thought it is rm.na